packages/mongo-nav/README.md

```diff
@@ -48,39 +48,39 @@ _Any other properties will be spread on the root element_
 
 #### Org Nav Elements
 
-`OrgNavAccessManager`  
-`OrgNavAdmin`  
-`OrgNavAllClusters`  
-`OrgNavBilling`  
-`OrgNavDropdownOrgAccessManager`  
+`OrgNavAccessManager`
+`OrgNavAdmin`
+`OrgNavAllClusters`
+`OrgNavBilling`
+`OrgNavDropdownOrgAccessManager`
```

Comment:
The trailing two spaces need to be preserved for all of these, to indicate a line break in markdown. We can replace them with `<br />` instead if needed, I think that github flavor markdown is okay with that.

Reply:
I'm not seeing this rendered any differently with or without the spaces. prettier prunes trailing whitespace, so going with prettier.